cleancat/base.py

```diff
@@ -687,7 +687,7 @@ def obj_to_dict(cls, obj):
         """
         data = {}
         for field_name in cls.get_fields():
-            if hasattr(obj, field_name):
+            if getattr(obj, field_name, None):
```

Comment:
First of all, something this would prevent calling `getattr` twice:

```suggestion
            value = getattr(obj, field_name, None)
            if value is not None:
```

Second problem though is: Do we want to distinguish between a `None` value, and no value at all?

Reply:
Very good points. See 1301f8e.